drivers/console/Kconfig.ws

```diff
@@ -0,0 +1,112 @@
+# Kconfig - console driver configuration options
+
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig WS_CONSOLE
+	bool "Enable websocket console service"
+	default n
+	select NETWORKING
+	select NET_TCP
+	select HTTP_PARSER
+	select NET_APP_SERVER
+	select WS
+	help
+	This option enables console over a websocket link. Currently,
+	it is basically just a redirection of the Zephyr console through
+	websocket. It nicely works along with another console driver (like
+	uart), twist being that it will take over the output only if a
+	successful connection to its HTTP service is done.
+
+if WS_CONSOLE
+
+config WS_CONSOLE_LINE_BUF_SIZE
+	int "WS console line buffer size"
+	default 128
+	help
+	This option can be used to modify the size of the buffer storing
+	console output line, prior to sending it through the network.
+	Of course an output line can be longer than such size, it just
+	means sending it will start as soon as it reaches this size.
+	It really depends on what type of output is expected.
+	If there is a lot of short lines, then lower this value. If there
```

Comment:
If there are ...

Reply:
:-(

> This contains also some patches that are already presented in other PRs

It didn't make sense to review WebSocket patches, as these are pulled in by a chance...

@jukkar : You know my solution to the problem - having smaller PRs with easier to review changes, and having quicker turnaround on them ;-). (And yep, I don't know why multi-face routing would depend on websocks...)

Reply:
> and having quicker turnaround on them

I completely forgot that it's Sep 6 and the release still wasn't made...